### ISSUE TITLE
Limited head rotation

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5941]
**Preferred term label:** Limited head rotation

**Synonyms**

**Definition (free text, please give PubMed ID)** Reduced range of motion turning the head side to side.

**Parent term (use hpo.jax.org/app)** Limited neck range of motion HP:0000466

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@JillMiller  I am wondering about the context of this term. Is it related to muscle weakness in neonatal neurological disease, i.e., to muscle weakness or is it related to other issues?

**Comment by pnrobinson** [comment]
adding term
